ocpmodels/trainers/base_trainer.py

```diff
@@ -317,6 +317,19 @@ def load_datasets(self) -> None:
                 if self.config["test_dataset"].get("use_train_settings", True):
                     test_config = self.config["dataset"].copy()
                     test_config.update(self.config["test_dataset"])
+                    # if a2g_args are used remove keys for labels
```

Comment:
@janiceblue and @mshuaibii have a look at these changes when you get a chance please.

@janiceblue can you try that this now works to predict by passing a test dataset in the command line?